backend/app/tests/crud/test_prompt_versions.py

```
@@ -0,0 +1,239 @@
+import pytest
+from fastapi import HTTPException
+from sqlmodel import Session
+
+from app.core.util import now
+from app.crud.prompt_versions import (
+    create_prompt_version,
+    delete_prompt_version,
+    get_next_prompt_version,
+)
+from app.crud.prompts import create_prompt
+from app.models import Prompt, PromptCreate, PromptVersion, PromptVersionCreate
```

Ruff (0.12.2)

`app.models.PromptCreate` imported but unused

Remove unused import: `app.models.PromptCreate`

(F401)